etc/example-config.yaml

```diff
@@ -35,3 +35,5 @@ broker:
   launch_apb_on_bind: false
   recovery: true
   output_request: true
+  ssl_cert_key: /path/to/key
```

Comment:
Should we default this to ```""```? dev_broker is being set to true, which means this should get defaulted to an empty string.